calcom/cal.com
# feat: ratelimit - ratelimit refactor + tests

This PR adds ratelimiting to booking page. Allowing 10/60s (we can reduce this if required but its better than no support)

This PR also adds a new helper function that throws and provides a detailed error message to the user about what went wrong if they hit the limit. 

How to test
* Add upstash env DEV keys from bitwarden
* try hit any of the rate limited APIs (try booking more than 10 times in 60 seconds)
* Expect error
